src/app/features/input/input.component.ts

```diff
@@ -53,17 +67,21 @@ export class InputComponent implements OnInit {
       return;
     }
 
-    if (!this.data.checkIfVideoIdIsOnTheList(dataToFetch.videoId)) {
+    if (this.checkIsVideoIdIsOnTheList(dataToFetch.videoId) === false) {
```

Comment:
!this.checkIsVideoIdIsOnTheList(dataToFetch.videoId)

Reply:
Tak, ale z jakiegoś powodu nie chciało mi się to aktywować w czasie testów i musiałem ominąć.